homeassistant/components/modbus/__init__.py

```diff
@@ -189,47 +194,59 @@ def name(self):
             await asyncio.sleep(self._config_delay)
             self._config_delay = 0
 
-    def setup(self):
-        """Set up pymodbus client."""
+    def _framer(self, method):
```

Comment:
This method may be static, consider adding the @staticmethod decorator.

Reply:
Done